lib/cunit/cu_pc_frustum.c

```diff
@@ -0,0 +1,167 @@
+		"010F0000800600000001030000800100000005000000000000000000F03F000000000000F03F000000000000F03F000000000000F03F0000000000000840000000000000F03F000000000000F03F000000000000F03F0000000000000840000000000000F03F00000000000008400000000000000840000000000000F03F000000000000F03F000000000000F03F010300008001000000050000000000000000000040000000000000F03F000000000000F03F00000000000000400000000000000840000000000000F03F0000000000000040000000000000F03F00000000000008400000000000000040000000000000084000000000000008400000000000000040000000000000F03F000000000000F03F01030000800100000005000000000000000000F03F000000000000F03F000000000000F03F0000000000000040000000000000F03F000000000000F03F000000000000F03F000000000000F03F00000000000008400000000000000040000000000000F03F0000000000000840000000000000F03F000000000000F03F000000000000F03F01030000800100000005000000000000000000F03F0000000000000840000000000000F03F00000000000000400000000000000840000000000000F03F000000000000F03F00000000000008400000000000000840000000000000004000000000000008400000000000000840000000000000F03F0000000000000840000000000000F03F01030000800100000005000000000000000000F03F000000000000F03F000000000000F03F0000000000000040000000000000F03F000000000000F03F000000000000F03F0000000000000840000000000000F03F00000000000000400000000000000840000000000000F03F000000000000F03F000000000000F03F000000000000F03F01030000800100000005000000000000000000F03F000000000000F03F00000000000008400000000000000040000000000000F03F0000000000000840000000000000F03F00000000000008400000000000000840000000000000004000000000000008400000000000000840000000000000F03F000000000000F03F0000000000000840"
+	};
+	for(i=0; i<N; ++i)
+		test_frustum_wkb_expected(f+i,expected[i]);
+}
+
+static void
+test_frustum_is_valid()
+{
+	int i;
+	for(i=0; i<N; ++i)
+		pc_frustum_is_valid(f+i);
+}
+
+static void
+test_frustum_intersects()
+{
+	int i, j, mode;
+	int expected[] = {
+		1,0,1,1,1,
+		0,1,0,0,0,
+		1,0,1,1,1,
+		1,0,1,1,1,
+		1,0,1,1,1,
+
+		1,0,1,1,1,
+		0,1,0,0,0,
+		1,0,1,1,1,
+		1,0,1,1,1,
+		1,0,1,1,1,
+
+		1,0,1,1,1,
+		0,1,0,0,0,
+		1,0,1,1,1,
+		1,0,1,1,1,
+		1,0,1,1,1,
+
+		1,0,1,1,1,
+		0,1,0,0,0,
+		1,0,1,1,1,
+		1,0,1,1,1,
+		1,0,1,1,1,
+	};
+	for(mode=1; mode<5; ++mode)
+		for(i=0; i<N; ++i)
```

Comment:
I am picky, but could we try respecting the styling used upstream?

```c
for ( mode = 1; mode < 5; ++mode )
{
    for ( i = 0; i < N; ++i)
    {
```

Feel free to reject me if you think I am too pedantic :)